docs/submitting-clinical-data.md

```diff
@@ -2,15 +2,28 @@
 id: submitting-clinical-data
 title: Submitting Clinical Data
 ---
+Clinical data consists of all supporting data about your programs donors and their health data.
```

Comment:
...about your program's donors

should this be more specific?

Reply:
yeah just placeholder for now